comp/etw/impl/etwSession.go

```diff
@@ -166,11 +166,18 @@ func (e *etwSession) StopTracing() error {
 		globalError = errors.Join(globalError, e.DisableProvider(guid))
 	}
 	ptp := (C.PEVENT_TRACE_PROPERTIES)(unsafe.Pointer(&e.propertiesBuf[0]))
-	ret := windows.Errno(C.ControlTraceW(
-		e.hSession,
-		nil,
-		ptp,
-		C.EVENT_TRACE_CONTROL_STOP))
+	var ret windows.Errno
+	if e.wellKnown {
+		ret = windows.Errno(C.CloseTrace(e.hTraceHandle))
```

Comment:
do we need to handle this error code case?

https://learn.microsoft.com/en-us/windows/win32/api/evntrace/nf-evntrace-closetrace
> If you call this function before ProcessTrace returns, the CloseTrace function returns ERROR_CTX_CLOSE_PENDING. The ERROR_CTX_CLOSE_PENDING code indicates that the CloseTrace function call was successful;

Reply:
no, because we're aware that the process trace might not return right away, and we block waiting for that to return.  and that error will be rolled up in the errors below (line 186)